### DISCUSSION TITLE
Could you support asciidoc?

### DISCUSSION THREAD

**Discussion by maojunxyz** [requarks/wiki#5662] [Help / Questions] [answered]
Similar to markdown but more powerful than markdown.

Official Website: https://asciidoc.org/

**Comment by NGPixel** [comment, marked as answer]
It's already planned for a future release.

"more powerful than markdown" is debatable, as you can achieve pretty much everything asciidoc offers in the markdown syntax of Wiki.js.

**Reply by luolong** [reply]
> "more powerful than markdown" is debatable

I would argue that text formatting wise, you may be correct. But that is just part of the story. 

A very powerful feature of asciidoc is ability to [include other documents (or document snippets) in your asciidoc](https://docs.asciidoctor.org/asciidoc/latest/directives/include/).

As far as I can tell, there is no equivalent feature in any flavor of Markdown that I have come across...
